modules/462-loki/openapi/doc-ru-config-values.yaml

```
@@ -19,7 +19,21 @@ properties:
 
       Занятый объем автоматически мониторится с помощью модуля [extended-monitoring](../extended-monitoring/configuration.html#namespaced-kubernetes-объекты)
 
-      **Внимание.**  Если диск с данными Loki переполнится, приложение перестанет работать. Вам следует выбрать необходимый баланс между значениями параметров `retentionPeriodHours` и `diskSizeGigabytes`.
+      **Внимание.**  Если диск с данными Loki переполнится, приложение перестанет работать. Вам следует либо выбрать необходимый баланс между значениями параметров `retentionPeriodHours` и `diskSizeGigabytes`,
+      либо использовать сочетание верхней границы занимаего чанками места `maxDiskUsageMegabytes` вместе с `maxDiskUsageCheckIntervalSeconds`.
+  maxDiskUsageMegabytes:
+    description: |
+      Это значение будет использоваться в качестве порогового для запуска процесса очистки от устаревших чанков Loki.
+      Значение по умолчанию: 0 (старые чанки не удаляются).
+  maxDiskUsageCheckIntervalSeconds:
+    description: |
+      Интервал для запуска Loki compaction, которому будет предшествовать хук принудительно маркировки чанков на удаление.
+      В процессе такого модифицированного компакшена (если значение `maxDiskUsageMegabytes` > 0),
+      хук проверит размер директории хранилища чанков `common.storage.filesystem.chunks_directory`.
```

```suggestion
      В процессе такой модифицированной оптимизации (если значение `maxDiskUsageMegabytes` > 0), хук проверит размер директории хранилища чанков `common.storage.filesystem.chunks_directory`.
```

Предлагаю «компакшена» заменить на «оптимизации», если не против. Или хотя бы на «компактизации».

---

Да, номально.

---

только предлагаю убрать слово модифицированной - вргоде как лишнее

---

А может все-таки оставить, чтобы сделать акцент на отличие от стандартного механизма оптимизации? Про него будет ниже.

---

Если есть отличия, то да, конечно. Давайте напишем чтобы было понятно что используется модифицированный механизм оптимизации.

---

НО с другой стороны - мы не пишем в чем суть модификации. Может в доке про это написать просто как описание фичи модуля?